src/app/loginComponents/requests/requests.component.html

```diff
@@ -26,11 +26,19 @@ <h3>Curator Requests</h3>
               on {{ org.dbCreateDate | date }}
             </p>
           </mat-card-subtitle>
+          <button
+            mat-flat-button
+            [id]="'delete-pending-org' + i"
```

Comment:
I see this pattern was already there, but it looks like we have this ids only for use by Cypress (?). If so, we should probably be using data-cy instead. That said, don't bother changing this, just keep it in mind for the future.